### DISCUSSION TITLE
Meeting Minutes 3/30/23

### DISCUSSION THREAD

**Discussion by Caleb-Cohen** [timmyichen/dev-directory#72] [Meeting Minutes]
## Agenda
- Week in review - what happened? What got shipped? 
  - #71 
  - #70 
  - #69 
  - #68 
  - #49 
  - #65 
- What's in-flight - what do people need to get unblocked? 
  - #58 
  - #59 
  - #60 
- The week ahead - 
  - Onboarding Tickets
  - @Natashka333 continued interviews
- Next meeting: @wo1vin to run things

## Parking Lot Items

**Friendly reminder from the group** - don't be afraid to ask for help! If you're stuck on something for longer than an hour, stop and ask to pair with someone. We're all friendly here and still learning. It's very likely that when you pair with someone, both of you will be learning something new. And it's fun! So lean into it often!

- Onboaring Tech Spec Page was decided over modal. Flags instead of updated User model. Single page instead of multi page. 

## Action Items

- @jmoldyvan will be adding issues for onboarding

**Comment by Natashka333** [comment]
Interview findings for now
<img width="675" alt="Screenshot 2023-03-30 at 6 52 57 PM" src="https://user-images.githubusercontent.com/102708144/228989537-7117a175-337b-4305-8a58-4d927fcc7031.png">
